beta-src/src/assets/css/index.css

```diff
@@ -1,3 +1,17 @@
-::-webkit-scrollbar {
+/* ::-webkit-scrollbar {
   width: 0;
+} */
+
+/* FIXME: I need a scrollbar shown for MessageList.
+idk how to specialize based on object class or whatever.
+*/
+::-webkit-scrollbar {
```

Comment:
I think we need to keep this as scrollbars were showing where they shouldn't be on chrome / brave on windows.

Reply:
Hmm... from what I can tell this *globally* removes scroll bars, and I don't see a way to do it in a component-specific way.

Is there any other way you can intervene to remove those scrollbars without disabling *all* scrollbars?

Reply:
@bryancodazen can you tell me how to reproduce "scrollbars showing where they shouldn't on chrome"? I made this change and have not seen any unexpected scrollbars on my chrome setup.

Reply:
Hey @adamlerer, I see scrollbars on the entire window (when the popover is closed) in Brave and Chrome (both are up to date) on windows PC.

Brave: 
![scrollbars-1](https://user-images.githubusercontent.com/93741108/168701427-c0e01432-363c-4518-8cb8-0330b8760f7a.png)

Chrome: 
![scrollbars-chrome-2](https://user-images.githubusercontent.com/93741108/168701446-a9c774dc-d186-4a95-9252-8ab150c011fa.png)
![scrollbars-chrome](https://user-images.githubusercontent.com/93741108/168701449-a3605a54-7752-40b1-9528-5ce90d3229fc.png)

Reply:
Thanks, I was able to repro after closing the modal. Not sure what the solution here is...

Reply:
@neillabsai Could you help here?